tests/scripts/all.sh

```diff
@@ -907,7 +907,25 @@ make test
 cd "$MBEDTLS_ROOT_DIR"
 rm -rf "$OUT_OF_SOURCE_DIR"
 
+msg "build: Default + MBEDTLS_SSL_OUT_CONTENT_LEN (ASan build)"
```

Comment:
Can we put a comment and break in here, to separate out the different phase of tests?